dashboard/src/main/webapp/package.json

```diff
@@ -30,6 +30,7 @@
   "dependencies": {
     "axios": "^1.4.0",
     "core-js": "^3.8.3",
+    "dayjs": "^1.11.10",
```

Comment:
Could you use `moment`? The `moment` module has been introduced.

Reply:
> Could you use `moment`? The `moment` module has been introduced.

The moment module has been adopted.